File: DesafioSistemaTarefas.Application/DTOs/TarefaDto.cs
using System.Diagnostics.CodeAnalysis;

namespace DesafioSistemaTarefas.Application.DTOs
{
    [ExcludeFromCodeCoverage]
    public class TarefaDto : BaseDto
    {
        public string nome { get; set; }
        public string descricao { get; set; }
        public DateTime dataHoraTarefa { get; set; }
        public int idStatusTarefa { get; set; }
        public string status { get; set; }
    }
}

File: DesafioSistemaTarefas.Front/src/app/tarefa/models/TarefaDto.ts
export class TarefaDto {

    public TarefaDto()
    {
        
    }
    id: number = 0;
    nome: string = '';
    descricao: string = '';
    idStatusTarefa: number = 0;
    status: string = '';
    dataHoraTarefa: Date = new Date();
    dataAtualizacao: Date = new Date();
    dataCriacao: Date = new Date();
}
